data/tlg0010/tlg011/tlg0010.tlg011.perseus-eng2.xml

```diff
@@ -27,16 +26,22 @@
       </publicationStmt>
 
       <sourceDesc>
-        <bibl>
-          <author>Isocrates</author>
-          <title>Isocrates with an English Translation in three volumes, by George Norlin, Ph.D.,
-            LL.D.</title>
-          <publisher>Cambridge, MA, Harvard University Press; London, William Heinemann
-            Ltd.</publisher>
-          <biblScope unit="volume">1</biblScope>
-          <date type="print">1928</date>
-          <date type="reprint">1980</date>
-        </bibl>
+        <biblStruct>
+          <monogr>
+            <author>Isocrates</author>
+            <title>Isocrates</title>
+            <editor role="translator">George Norlin</editor>
+            <imprint>             
+              <pubPlace>Cambridge, MA</pubPlace>
+              <pubPlace>London, UK</pubPlace>
```

Comment:
I don't think UK is included anywhere else?

Reply:
I'm not sure, I will do a global check at the end.